service/src/main/java/uk/gov/hmcts/reform/fpl/config/AsyncConfiguration.java

```diff
@@ -18,11 +18,9 @@ public AsyncUncaughtExceptionHandler getAsyncUncaughtExceptionHandler() {
 
     public class AsyncExceptionHandler implements AsyncUncaughtExceptionHandler {
 
-        private final Logger logger = LoggerFactory.getLogger(getClass());
-
         @Override
         public void handleUncaughtException(Throwable throwable, Method method, Object... obj) {
-            logger.error("Unexpected error occurred during async execution", throwable);
+            log.error("Unexpected error occurred during async execution", throwable);
```

Comment:
any need using the `com.microsoft.applicationinsights` library to also log the errors to appinsights?

Reply:
No, it does not work this way. App insight is automatically integrated with standard loggers so all logs lands in azure
https://github.com/Microsoft/ApplicationInsights-Java/tree/v2.3.1/azure-application-insights-spring-boot-starter

Reply:
thanks for the link @tomaszpowroznik.